hugo/content/info/corrections.md

```diff
@@ -47,6 +47,7 @@ A request to change paper metadata (that does not require any PDF changes) can b
 
 The Anthology team will attend to the correction as we find time.
 Metadata changes are generally accepted if they are consistent with the PDF, which we take as authoritative.
+However, please see the note below about author changes.
```

Comment:
Can be linked:
`However, please see the [note below about author changes](#note-on-author-changes).`